docs/rapha-rover/specification/ros-api.mdx

```diff
@@ -0,0 +1,567 @@
+---
+title: ROS API
+sidebar_label: ROS API
+---
+
```

Comment:
Please add keywords and description fields to the front matter.

Reply:
The `title` field is also used by search engines. Currently not specified which robot the ROS API belongs to. I suggest setting a custom H1 and specifying the title as below: 

```suggestion
---
title: Rapha Rover ROS API
sidebar_label: ROS API
---

# ROS API
```

Reply:
done